src/main/java/com/nike/cerberus/service/AuthenticationService.java

```diff
@@ -224,6 +224,18 @@ public IamRoleAuthResponse authenticate(IamPrincipalCredentials credentials) {
         return authenticate(credentials, authPrincipalMetadata);
     }
 
+    /**
+     * Enables an IAM role to authenticate and get back an UNENCRYPTED payload
+     * @param iamPrincipalArn IAM role ARN
+     * @return Unencrypted auth response
+     */
+    public AuthTokenResponse keylessAuthenticate(final String iamPrincipalArn) {
```

Comment:
instead of keyless auth, how about stsAuth?  or some other phrase describing "what it is" instead of saying "what it is not".

Reply:
good point